dev-libs/expat/expat-2.5.0.recipe

```diff
@@ -6,8 +6,8 @@ HOMEPAGE="http://expat.sourceforge.net/"
 COPYRIGHT="1998-2000 Thai Open Source Software Center Ltd and Clark Cooper
 	2001-2022 Expat maintainers."
 LICENSE="MIT"
-REVISION="3"
-SOURCE_URI="https://downloads.sourceforge.net/expat/expat-$portVersion.tar.bz2"
+REVISION="4"
```

Comment:
Changing the URI should be enough here if there is no real reason to rebuild it, this would keep the legacy version currently in the depot.